composer.json

```diff
@@ -48,7 +48,11 @@
         "doctrine/orm": "The Doctrine ORM integration is optional in the bundle."
     },
     "autoload": {
-        "psr-4": { "Doctrine\\Bundle\\DoctrineBundle\\": "" }
+        "psr-4": { "Doctrine\\Bundle\\DoctrineBundle\\": "" },
+        "exclude-from-classmap": ["Tests/"]
```

Comment:
@Ocramius we will also need to remember updating the autoloading config each time we add a new top-level subnamespace or class, which is a pain. This adds maintenance overhead for the future.

This exclusion is just about optimizing the classmap of the optimized autoloader, to avoid including tests in it. And this is because we have tests inside a subfolder of the runtime code.

If we want to avoid using this feature, I would rather reorganize the repo to use a proper package layout (with ``src/`` and ``tests`` at the root, and most of the existing root content inside ``src``) instead of using the bundle layout which was created at a time before PSr-4 was a thing (and so a package layout would have been a pain with a deep nested folder)